ONSdigital/dp-frontend-search-controller
# RSS feed functionality for data aggregation pages

### What

Add rss feed functionality for data aggregation pages
Trello: https://trello.com/c/CMXt83iZ

### How to review

To test the rss feed locally, you can 

- run dp ssh sandbox web 1 -p 23200:localhost:10800
- set EnableReworkedDataAggregationPages config flag to true
- Then run make debug on this branch. 
- Navigate to http://localhost:25000/alladhocs?rss
- All of the other data aggregation pages should work with the that rss, as well as other query params

### Who can review

Anyone
